rpc/src/client/transport/websocket.rs

```diff
@@ -52,8 +56,13 @@ const RECV_TIMEOUT: Duration = Duration::from_secs(RECV_TIMEOUT_SECONDS);
 // Taken from https://github.com/tendermint/tendermint/blob/309e29c245a01825fc9630103311fd04de99fa5e/rpc/jsonrpc/server/ws_handler.go#L28
 const PING_INTERVAL: Duration = Duration::from_secs((RECV_TIMEOUT_SECONDS * 9) / 10);
 
-/// Low-level WebSocket configuration
-pub use async_tungstenite::tungstenite::protocol::WebSocketConfig;
+#[derive(Default, Debug)]
+/// WebSocket client configuration
+pub struct WebSocketConfig {
+    pub compat: CompatMode,
+    /// Low-level protocol configuration
+    pub transport: Option<async_tungstenite::tungstenite::protocol::WebSocketConfig>,
+}
```

Comment:
Also a builder would be better here, instead of introducing another struct with public members.

Reply:
Redone in https://github.com/informalsystems/tendermint-rs/pull/1193/commits/84d9cad8c8e41e9e0343633a06ae8288101b0ad3